fuzz/run.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+handle_fail()
+{
+  let "fails++"; mkdir "fail-${fails}" ; sed -i "1iFailed instance number ${fails}, with password '$password'" out ; cat out ; cat out >> fails ; mv out "fail-${fails}" ;
+}>&2
+
+fails=0
+runs=0
+echo -n "" > fails
+while true
+ do
+   password=""
+   radamsa test_mnemonic_str > mnemonic_str
+   radamsa test_entropy_bin > entropy_bin
+   radamsa test_entropy_hex > entropy_hex
+   radamsa test_seed_bin > seed_bin
+   radamsa test_seed_hex > seed_hex
+
+   mnemoniccli -g -e entropy_hex -s seed -m mnemonic -ll debug &>out
+   if [[ $? != 0 && $? != 65 ]] ; then handle_fail ; cp entropy_hex "fail-${fails}" ; fi
```

Comment:
`$?` gives you exit code of last command, so it's better to read value once into variable and then do comparison on several places. Variable stays the same, `$?` changes if you execute another command in the meantime.

Reply:
the comparison is done only in one place for each run of mnemoniccli, 'if [[ ... ]]' is a single command as far as I know

Reply:
I'm afraid this is issue like "it works in this case, but different approach is better in general" :D Do you want to fix this or should I mark it as *wontfix*?

Reply:
No, mark it as false positive instead, it's not that it only works in this case. Codeclimate wants us not to use '$?' variable at all, which cant be done in this case.

Reply:
Shellcheck (code climate) is fine with approach I suggested as solution. It does not have problem with $?.

Reply:
then you found a way around its checks, you still use indirect check ($?) not direct, which is what was reported

Reply:
>* Since the command and its status test are decoupled, inserting an innocent command like `echo "make finished"` after `make` will cause the `if` statement to silently start comparing `echo`'s status instead.
> * The value of `$?` is overwritten by `[`/`[[`, so you can't get the original value in the relevant then/else block (e.g. `if mycmd; then echo "Success"; else echo "Failed with $?"; fi`).
> https://github.com/koalaman/shellcheck/wiki/SC2181